rancher_pr_deploy.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+DEPENDANT_SERVICE=$1
```

Comment:
The dependent services can be inferred from looking at the docker-compose.yml file but, as it stands, the differences in naming makes it non-trivial (e.g. c2s-frontend vs connecting-to-services) so this approach works for now. The variety of names we use for the same things is making it harder for us to script stuff.